revsolver/base/openshift-templates/revsolve_and_sync-template.yaml

```diff
@@ -0,0 +1,116 @@
+apiVersion: template.openshift.io/v1
+kind: Template
+metadata:
+  name: revsolver
+  annotations:
+    description: "Thoth: Reverse Solver  Template"
+    openshift.io/display-name: "Thoth: Reverse Solver"
+    version: 0.6.0
```

Comment:
let us remove the version so that argo would maintain it.

Reply:
do'h always forgot it. Thanks for spotting this :+1: